later/lib/widgets/buttons/primary_button.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+
+class PrimaryButton extends StatelessWidget {
+  const PrimaryButton({
+    super.key,
+    required this.onTap,
```

Comment:
Actually wait: if it is nullable, then it shouldn't be `required`.

```suggestion
    this.onTap,
```